pkg/pillar/cmd/diag/diag.go

```diff
@@ -143,7 +143,7 @@ func Run() {
 		DeviceNetworkStatus: ctx.DeviceNetworkStatus,
 		FailureFunc:         zedcloud.ZedCloudFailure,
 		SuccessFunc:         zedcloud.ZedCloudSuccess,
-		NetworkSendTimeout:  ctx.globalConfig.NetworkTestTimeout,
+		NetworkSendTimeout:  types.GetGlobalValue(types.NetworkSendTimeout, ctx.configItemValueMap).IntValue,
```

Comment:
Same as above.. 

ctx.configItemValueMap.ItemValue(<key>)

As an aside, configItemValueMap --> Rename this as globalConfig ??

NetworkSendTimeout: ctx.globalConfig.ItemValue(<key>)